common/history/states/african_states.txt

```diff
@@ -480,7 +464,7 @@ STATES = {
 			}
 		}
 		
-		add_claim = c:ETH add_homeland = ethiopian
+		add_claim = c:ETH add_homeland = ethiopian add_homeland = oromo add_homeland = somali
```

Comment:
Please change it to this:
add_claim = c:ORO add_homeland = oromo 
add_claim = c:SOM add_homeland = somali